controllers/compaction/reconciler.go

```diff
@@ -348,13 +348,18 @@ func getCompactionJobVolumeMounts(etcd *druidv1alpha1.Etcd, logger logr.Logger)
 		logger.Error(err, "Storage provider is not recognized. Compaction job will not mount any volume with provider specific credentials", "namespace", etcd.Namespace, "name", etcd.Name)
 		return vms
 	}
-
-	if provider == utils.GCS {
+	switch provider {
+	case utils.Local:
+		vms = append(vms, v1.VolumeMount{
+			Name:      "host-storage",
+			MountPath: pointer.StringDeref(etcd.Spec.Backup.Store.Container, ""),
+		})
```

Comment:
When using EtcdWrapper, special handling is required to mount the volume to nonroot directory. Please refer to https://github.com/gardener/etcd-druid/blob/0477a279db70d42a83cd94fa0de8caad7b70a990/controllers/etcdcopybackupstask/reconciler.go#L476-L487 as an example

Reply:
We  won't need it because https://github.com/gardener/etcd-druid/pull/682#issuecomment-1744249958